content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
+| Availability          | The feature may or may not be available in the main Istio repository. The feature may or may not appear in an Istio release. If it does appear in an Istio release it will be disabled by default. The feature requires an explicit flag to enable in addition to any configuration required to use the feature, in order to turn on dev features.                                                                                                                         | The feature is committed to the Istio repository The feature appears in an official Istio release The feature requires explicit user action to use, e.g. a flag flip, use of a config resource, an explicit installation action, or an API being called. When a feature is disabled it must not affect system stability.                                                                                                                           | In official Istio releases Enabled by default                                                                                                                                                                                                                                                                                                                                                                                                                                               | Same as Beta                                                                                                                               |
```

Comment:
```suggestion
| Availability          | The feature may or may not be available in the main Istio repository. The feature may or may not appear in an Istio release. If it does appear in an Istio release it will be disabled by default. The feature requires an explicit flag to enable in addition to any configuration required to use the feature, in order to turn on dev features.                                                                                                                         | The feature is committed to the Istio repository. The feature appears in an official Istio release. The feature requires explicit user action to use, e.g. a flag flip, use of a config resource, an explicit installation action, or an API being called. When a feature is disabled it must not affect system stability.                                                                                                                           | In official Istio releases Enabled by default                                                                                                                                                                                                                                                                                                                                                                                                                                               | Same as Beta                                                                                                                               |
```